apps/admin/frontend/test/helpers/mock_api_client.ts

```diff
@@ -96,9 +97,9 @@ export function createMockApiClient(): MockApiClient {
  * })
  *
  * userEvent.click(screen.getButton('Print'));
- * await screen.findByText('Printing');
+ * await vi.waitFor(() => screen.get)ByText('Printing');
  * resolve();
- * await screen.findByText('Printed');
+ * await vi.waitFor(() => screen.get)ByText('Printed');
```

Comment:
I think these comments were inadvertently edited by a find-and-replace

Reply:
Fixed.